### PR TITLE
[stable/prometheus-operator] add config to enable or disable rules in kube-apiserver-error.yaml

### PR DESCRIPTION
**Pull request by qingkunl**
No

#### What this PR does / why we need it:

This PR adds a config value to enable or disable rules in kube-apiserver-error.yaml (default to enable). This makes it consistent with all the other rules, which all have such an enable value.

#### Which issue this PR fixes

N/A

#### Special notes for your reviewer:

Have locally tested with helm lint and install

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by sev3ryn** [comment]
@bismarck @gianrubio  what else is needed for this to be merged?

**Comment by qingkunl** [comment]
force-push to bump the chart version, as it has been bumped by other commits since the last submit.

**Comment by qingkunl** [comment]
Could someone help with the test failure? Not sure if it's caused by the PR change or the testing environment.

> 1043
I0206 19:12:57.861]   Normal   NotTriggerScaleUp  5s (x2 over 15s)  cluster-autoscaler  pod didn't trigger scale-up (it wouldn't fit if a new node is added): 1 node(s) didn't have free ports for the requested pod ports, 1 node(s) didn't match node selector
1044
I0206 19:12:57.862]   Warning  FailedScheduling   2s (x4 over 17s)  default-scheduler   0/3 nodes are available: 2 node(s) didn't match node selector, 3 node(s) didn't have free ports for the requested pod ports.

**Comment by sev3ryn** [comment]
@qingkunl try to `/retest` looks like it is CI issue

File: stable/prometheus-operator/hack/sync_prometheus_rules.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def new_representer(dumper, data):
     'general.rules': ' .Values.defaultRules.rules.general',
     'k8s.rules': ' .Values.defaultRules.rules.k8s',
     'kube-apiserver.rules': ' .Values.kubeApiServer.enabled .Values.defaultRules.rules.kubeApiserver',
+    'kube-apiserver-error': ' .Values.kubeApiServer.enabled .Values.defaultRules.rules.kubeApiserverError',
```
Comment (sev3ryn) [inline stable/prometheus-operator/hack/sync_prometheus_rules.py:57]: Shouldn't here be kube-apiserver-error.rules ?

Comment (qingkunl) [reply stable/prometheus-operator/hack/sync_prometheus_rules.py:57]: I think the name here should follow the corresponding rule file name:
kube-apiserver.rules.yaml -> kube-apiserver.rules
kube-apiserver-error.yaml -> kube-apiserver-error

So if we'd like to use 'kube-apiserver-error.rules', the rule file should be renamed to kube-apiserver-error.rules.yaml.

This is my understanding, though. Let me know if that's incorrect.

**Comment by sev3ryn** [comment]
I see same build issue for all other opened [stable/prometheus-operator] PRs. So probably - just need to wait when it will be fixed

**Comment by qingkunl** [comment]
This CI issue has existed for a few days. Is this being worked on, and is there an issue open to track it?

**Pull request merged**